Dockerfile

```diff
@@ -1,12 +1,21 @@
-FROM registry.ci.openshift.org/open-cluster-management/builder:nodejs14-linux-amd64
-USER root
-RUN yum -y remove nodejs-nodemon
-RUN yum -y update
-USER default
+# Copyright Contributors to the Open Cluster Management project
 
-# Root needed for yum update, gets reset to 1001 below.
+FROM registry.ci.openshift.org/open-cluster-management/builder:nodejs14-linux-amd64 as builder
```

Comment:
Can we keep that comment? "Root needed for yum update"? One day there will be a question, and the comment will save us time in remembering why

Reply:
Actually, we aren't doing `yum update` anymore because we changed the base image. I've removed the root user.

Reply:
Verified PR image on my cluster after removing `USER root`.
2.2.0-PR98-f9dce188519df55571619dd80bb80b1cee0263e1